internal/app/config.go

```diff
@@ -5,19 +5,18 @@ import (
 )
 
 type Configuration struct {
-	ListenAddress     string              `mapstructure:"listen_address"`
-	DeveloperMode     bool                `mapstructure:"developer_mode"`
-	JWTAuth           []ginjwt.AuthConfig `mapstructure:"ginjwt_auth"`
-	FleetDBAPIOptions FleetDBAPIOptions   `mapstructure:"fleetdb"`
+	ListenAddress string              `mapstructure:"listen_address"`
+	DeveloperMode bool                `mapstructure:"developer_mode"`
+	JWTAuth       []ginjwt.AuthConfig `mapstructure:"ginjwt_auth"`
+	FleetDBOpts   FleetDBAPIOptions   `mapstructure:"fleetdb"`
 }
 
 // https://github.com/metal-toolbox/fleetdb
 type FleetDBAPIOptions struct {
-	Endpoint             string   `mapstructure:"endpoint"`
-	OidcIssuerEndpoint   string   `mapstructure:"oidc_issuer_endpoint"`
```

Comment:
Do we still want to keep `internal/fleetdb/fleetdb.go` and keep `OidcIssuerEndpoint`?
I was duplicating client initialization code from the [conditionorc repo](https://github.com/metal-toolbox/conditionorc/blob/main/internal/fleetdb/interface.go#L83) which uses  `OidcIssuerEndpoint` as oidc provider